.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,31 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/setup-r@v1
+      - name: Install dependencies
+        run: |
+          install.packages(c("remotes", "rcmdcheck"))
+          remotes::install_deps(dependencies = TRUE)
```

Comment:
works fine with `remotes::install_deps(dependencies = TRUE, repos = BiocManager::repositories())`.

Using this, the changes in the DESCRIPTION file shouldn't be necessary

Reply:
Done, `README.md` updated to reflect it.